adam-core/src/main/scala/org/bdgenomics/adam/models/ReferenceRegion.scala

```diff
@@ -503,19 +605,53 @@ case class ReferenceRegion(
   }
 
   /**
+   * Determines if two regions are colocated on the same strand.
+   *
+   * @param other The other region.
+   * @return True if the two are on the same strand, false otherwise
+   */
+  def sameStrand(other: ReferenceRegion): Boolean = {
```

Comment:
It was changed to this for readability. Won't the compiler just replace the method call with the equality statement inline anyway?